src/pcap/test/mocks.go

```diff
@@ -0,0 +1,320 @@
+package test
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"regexp"
+	"text/template"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+func MustParseURL(rawURL string) *url.URL {
+	url, err := url.Parse(rawURL)
+	if err != nil {
+		panic(err)
+	}
+	return url
+}
+
+func MockJwtAPI() (*httptest.Server, string) {
```

Comment:
If you wanted to keep the Mock Bosh Director around as executable, it should share code with this.